Assignment-1/models/item.go

```diff
@@ -0,0 +1,17 @@
+package models
+
+type Item struct {
+	Name     string
+	Price    float64
+	ItemType string
+}
+
+func (o *Item) Getter() Item {
```

Comment:
this can be done in single print statement.

Reply:
fixed it